src/views/RecIM/views/Activity/index.js

```diff
@@ -15,17 +17,24 @@ import { useUser } from 'hooks';
 import GordonLoader from 'components/Loader';
 import GordonUnauthorized from 'components/GordonUnauthorized';
 import styles from './Activity.module.css';
-import { MatchList, TeamList } from './../../components/List';
+import { MatchList, SeriesList, TeamList } from './../../components/List';
+import ActivityForm from 'views/RecIM/components/Forms/ActivityForm';
 import CreateTeamForm from '../../components/Forms/CreateTeamForm';
 import { getActivityByID } from 'services/recim/activity';
 import { Link as LinkRouter } from 'react-router-dom';
+import CreateSeriesForm from 'views/RecIM/components/Forms/CreateSeriesForm';
 
 const Activity = () => {
   const { activityID } = useParams();
   const { profile } = useUser();
   const [loading, setLoading] = useState(true);
   const [activity, setActivity] = useState({});
+  const [openActivityForm, setOpenActivityForm] = useState(false);
   const [openCreateTeamForm, setOpenCreateTeamForm] = useState(false);
+  const [openCreateSeriesForm, setOpenCreateSeriesForm] = useState(false);
+  const subElementStyle = {
```

Comment:
If you are leaving this here until Cam can figure out a fix for this then that is fine.

Reply:
I handled this in the next PR